aten/src/ATen/native/transformers/hip/flash_attn/ck/mha_fwd_ck.hip

```diff
@@ -0,0 +1,359 @@
+/******************************************************************************
+ * Copyright (c) 2024, Tri Dao.
```

Comment:
If those kernels are copy-n-paste from https://github.com/Dao-AILab/flash-attention imo it would be nice to mention in PR description,  modify respective README/LICENSE files and mention in PR that they it's co-authored by @tridao

Reply:
Ah, they are from there. and you're right